cg/models/demultiplex/flow_cell.py

```diff
@@ -197,10 +197,21 @@ def validate_sample_sheet(self) -> bool:
 
     def get_sample_sheet(self) -> SampleSheet:
         """Return sample sheet object."""
-        return get_sample_sheet_from_file(
-            infile=self.sample_sheet_path,
-            flow_cell_sample_type=self.sample_type,
-        )
+        try:
+            return get_sample_sheet_from_file(
+                infile=self.sample_sheet_path,
+                flow_cell_sample_type=self.sample_type,
```

Comment:
Are you testing in stage? It should fail because most of the flow cells in stage have some sample sheet missing but you can create them with `cg demultiplex samplesheet create-all` and it will work then.

Reply:
Ah, thanks for the tip! That was an issue for sure